monty_hall/monty_hall.py

```diff
@@ -0,0 +1,31 @@
+import random
+
+SWITCH = True
+goats = 0
+boats = 0
+runs = 1000000
+
+for _ in range(runs):
```

Comment:
we have not covered default / don't care values (_) might be worth just putting the variable run